pkg/filesystem/config.go

```diff
@@ -68,3 +80,10 @@ func WithEnableStargz(enable bool) NewFSOpt {
 		return nil
 	}
 }
+
+func WithEnableReferrers(enable bool) NewFSOpt {
+	return func(fs *Filesystem) error {
+		fs.enableReferrers = enable
```

Comment:
Can we just use `fs.referrerMgr` to judge if the referrer is enabled?

Reply:
Fixed